### ISSUE TITLE
Accessibility audit - headings structure

### ISSUE THREAD

**Issue by katarinazaprazna** [bratislava/olo.sk#819]
accessibility audit PDF:
https://inovaciebratislava.slack.com/archives/C06FW05TXHC/p1730379593029059

h1 on Homepage
missing h2, h1 on other subpages

Also address: [Page should contain a level-one heading](url) (axe DevTools audit)

![Image](https://github.com/user-attachments/assets/e5b851c0-bf98-4c70-9354-1e4455201a01)

**Comment by jankaBehal** [comment]
h1 on homepage merged
find out where are missing h1 on other pages
find solutioni with h2 missing, maybe discussion needed
